Enterprise-CMCS/macpro-mdct-mfp
# Mdct 2746 - add sidebar component

### Description
Added the sidebar component at pretty much the most basic level. No routing, no ReportProvider. I've set isHidden to false so you should see the sidebar appear when you go to /standard


### Related ticket(s)


MDCT-2764

---
### How to test
spin up locally, go to /standard, see the sidebar

![Screenshot 2023-08-10 at 9 42 12 AM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/22454493/380bcc63-b290-4657-8b5c-80e47eca1931)


### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
